defaults/main.yml

```diff
@@ -1,2 +1,16 @@
-wipefs_become: true
-wipefs_become_user: root
+# Whether or not to use the `become` feature of Ansible to gain admin
+# privileges.
+disk_wipe_become: true
+
+# Whether or not to use the `become` feature of Ansible to gain admin
+# privileges.
+disk_wipe_become_user: root
+
+# List of disks to run the role on.
+disk_wipe_disks: []
+
+# Example
```

Comment:
Given that this role is so highly dangerous, I think the examples in the README are sufficient for documentation, and listing common disk paths here, even commented out, could lead to a user typo that could create data loss.  Best to leave this out in my opinion.

"List of paths to disk block devices" would serve as a sufficient comment.

Reply:
updated